client/client.go

```diff
@@ -707,12 +707,12 @@ func (c *client) UpdateOption(option DynamicOption, value interface{}) error {
 			return errors.New("[pd] invalid value type for EnableTSOFollowerProxy option, it should be bool")
 		}
 		c.option.setEnableTSOFollowerProxy(enable)
-	case EnableFollowerHandle:
+	case EnableFollowerHandleRegionRequest:
```

Comment:
If we have more and more requests, will the option become bloated in the future?

Reply:
It'll be more, but I don't think it'll be bloated. If there are more options, we can provide a function to turn all options on or off, but we must be able to control them separately